frontend-maven-plugin/src/it/pnpm-integration/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "example",
+  "version": "0.0.1",
+  "dependencies": {
+    "less": "~3.0.2"
```

Comment:
Why is the `less` package needed? It's a CSS pre-processor. Not sure how it's related to `pnpm`?

Reply:
Copied from npm. I only touched the java files. But I will remove it.

Reply:
https://github.com/eirslett/frontend-maven-plugin/blob/3723ff74eea21a6dfbc4f184feaba1392f58808f/frontend-maven-plugin/src/it/node-provided-npm/package.json#L1-L10

I think the idea is it should actually fetch something so we see whether it actually works or not.